lib/apiservers/portlayer/swagger.json

```diff
@@ -1411,6 +1411,137 @@
 				}
 			}
 		},
+		"/interaction/{id}/serial": {
+			"post": {
+				"description": "Add serial port",
```

Comment:
While this is an internal API for now it is supposed to be exposed at some point - I don't think this is a level of detail we should be exposing. The caller of this API should know little to nothing about the infrastructure so how are they supposed to determine which are available or desired.
If we have different end points for interaction it should be because they provide different user facing semantics.